webui/apps/website/lib/dtos.ts

```diff
@@ -27,32 +22,25 @@ export interface Event {
    */
   description: string;
   /**
-   * three to five possible outcomes that are influenced by the location of this event
+   * Three possible choices to take in reaction to this event, which will dictate the next event in the world
    */
-  possible_outcomes: Outcome[];
+  possible_choices: Choice[];
 }
 /**
- * Outcome is the likely outcome from an Event, taking into account
- * recent developments in obscure wars, as we believe their impact
- * is more nuanced and far reaching than the government realizes.
+ * Choice is a possible choice to make in response to an Event
+ * This choice will have an impact on the world and how the story progresses
+ * It could lead to the world ending sooner, or later
  */
-export interface Outcome {
+export interface Choice {
   /**
-   * An unlikely but scary outcome description, one we didn't see coming
+   * The choice to make in response to the event
    */
-  description: string;
+  choice: string;
   /**
-   * The price that must be paid for progress. This is casualties, deaths, or other tragedies.
+   * The outcome of the choice made in response to the event
    */
   consequence: string;
-  /**
-   * A List of choices; only one of them will keep the world from ending
-   */
-  choices: string[];
-  /**
-   * A list of three outcomes; two are world ending outcomes and one keeps the world going
-   */
-  outcomes: string[];
+  [k: string]: unknown;
```

Comment:
hmm, so much for strong typing :(

Reply:
I don't even know if we truly need this prop, it also was messing up and making a `Choice1` type.